addons/udes_stock_package_move/tests/test_change_package_location.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from odoo.exceptions import UserError
+from addons.udes_stock.tests import common
+
+
+class TestChangePackageLocation(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestChangePackageLocation, cls).setUpClass()
+
+        Package = cls.env["stock.quant.package"]
+
+        cls.empty_package = Package.create({})
+        cls.package_1 = cls.create_package(cls.test_stock_location_01)
+        cls.package_2 = cls.create_package(cls.test_stock_location_01)
+        cls.package_3 = cls.create_package(cls.test_stock_location_01)
+        cls.package_4 = cls.create_package(cls.test_stock_location_02)
+        cls.packages = cls.package_1 | cls.package_2 | cls.package_3 | cls.package_4
+
+    @classmethod
+    def create_package(cls, location):
+        """
+        Create a package containing apples and bananas at the given location
+        """
+        Package = cls.env["stock.quant.package"]
+
+        package = Package.create({})
+        cls.create_quant(cls.apple.id, location.id, 10, package_id=package.id)
+        cls.create_quant(cls.banana.id, location.id, 10, package_id=package.id)
+        return package
+
+    def complete_picking(self, picking):
```

Comment:
Wondering if we want this one in `common.BaseUDES`, since this one will be needed for other modules.

What do you think about making the `action_done` optional, with a parameter `validate=true` by default ?